config/semi_automated.yml

```diff
@@ -461,6 +461,9 @@ parameters:
             regex: '\(\[\[User:Galobtter\/Shortdesc helper\|Shortdesc helper\]\]\)'
             link: User:Galobtter/Shortdesc helper
             namespaces: [0]
+        Undo-last-edit:
```

Comment:
should be lowercased, according to https://en.wikipedia.org/wiki/User:Enterprisey/undo-last-edit

Reply:
Done